index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Also one thing left to be done is changing your `readme.md` file. I would recommend changing the project name to Hello Microverse, adding some description change screenshot to your website screenshot add your name and social media links as so on 🍀👋

Reply:
Alright, I will do that now. Thanks